app/views/publish_document/published.html.erb

```diff
@@ -26,11 +26,16 @@
 
       <%= render "govuk_publishing_components/components/govspeak" do %>
         <%= govspeak_to_html t("publish_document.published.published_without_review.body_govspeak", title: @document.title) %>
+        <% stripped_url = DocumentUrl.new(@document).public_url.sub(/^https?\:\/\//,'') %>
+        <%= link_to(nil, stripped_url, class: "govuk-link govuk-link--no-visited-state") %>
 
-        <p> <%= t("publish_document.published.published_without_review.send_label") %> </p>
-
-        <%= link_to(nil, document_url(@document, utm_content: "2i-link"), class: "govuk-link") %>
+        <h2 id="what-to-do-next">What to do next</h2>
```

Comment:
Probably want to put this in a translation file, also not sure we need the id - I don't think we're anticipating people to deep link to ti.

Reply:
Cool, I'll do that